docs/modules/datasources/pages/performance_datasource.adoc

```diff
@@ -132,152 +126,166 @@ Consult the Measurements API documentation of your OpenNMS version for more deta
 
 === Filter queries
 
-Used to create new and/or alter existing series.
+Filter queries are used to create new series, or alter existing ones.
+The options provided depend on the filter that you select.
 
-The option provided depend on the selected filter.
-
-==== Outlier filter
+[caption=]
+.Filter query options
+[options="autowidth"]
+|===
+| Filter Name | Description
 
-Used to remove datapoints from the series that fall outside of the configured quantile, defaults to 95%.
-The datapoints are replaced with interpolated values.
+| Outlier
+| Removes datapoints that fall outside of the set quantile.
+Defaults to 95%. +
+The removed datapoints are replaced with interpolated values.
+|===
 
 [[ds-perf-template]]
 == Template queries
 
-Use template queries in metric queries and panel titles.
-The results will appear as a drop-down list at the top of the dashboard.
-You can then select from the list to change the data the panel displays.
-See xref:panel_configuration:dynamic-dashboard.adoc#pc-template-filters[Using Template Variables to Create Dynamic Dashboards] for an example.
+You can use template queries in metric queries and panel titles.
+The results will appear as a drop-down list at the top of the dashboard; you can select from the list to change the data that the associated panel displays.
+For an example of a template query, see <<panel_configuration:dynamic-dashboard.adoc#pc-template-filters, Use variables to create dynamic dashboards>>.
 
 === locations()
 
 The `locations()` query returns a list of all available node locations.
-Use this inside `nodeFilter()` to filter the number of nodes displayed by location.
+You can use this inside a `nodeFilter()` query to filter the number of nodes displayed by location.
 
-.Example variable expression
-* nodeFilter(location='Default')
-* nodeFilter(location='$location')
+.Example expressions using `locations()`
```

Comment:
These examples don't actually show the use of `locations()` in practice. These are examples of a `nodeFilter` that happens to filter on location.

Reply:
```suggestion
.Example expressions using `nodeFilter()`
```